tools/annotation_tools/turk_with_s3/create_jobs.py

```diff
@@ -8,49 +8,58 @@
 from urllib.parse import quote
 from datetime import datetime
 
-MTURK_SANDBOX = "https://mturk-requester-sandbox.us-east-1.amazonaws.com"
-access_key = os.getenv("AWS_ACCESS_KEY_ID")
-secret_key = os.getenv("AWS_SECRET_ACCESS_KEY")
-mturk = boto3.client(
-    "mturk",
-    aws_access_key_id=access_key,
-    aws_secret_access_key=secret_key,
-    region_name="us-east-1",
-    endpoint_url=MTURK_SANDBOX,
-)
-print("I have $" + mturk.get_account_balance()["AvailableBalance"] + " in my Sandbox account")
-
-
-def create_turk_job(xml_file_path: str, tool_num: int, input_csv: str):
-    # Delete HITs
-    # For use in dev only
-    for item in mturk.list_hits()["HITs"]:
-        hit_id = item["HITId"]
-        print("HITId:", hit_id)
-
-        # Get HIT status
-        status = mturk.get_hit(HITId=hit_id)["HIT"]["HITStatus"]
-        print("HITStatus:", status)
-
-        # If HIT is active then set it to expire immediately
-        if status == "Assignable" or status == "Reviewable":
-            response = mturk.update_expiration_for_hit(HITId=hit_id, ExpireAt=datetime(2015, 1, 1))
-
-        # Delete the HIT
-        try:
-            mturk.delete_hit(HITId=hit_id)
-        except:
-            print("Not deleted")
-        else:
-            print("Deleted")
+def create_turk_job(xml_file_path: str, tool_num: int, input_csv: str, job_spec_csv: str, use_sandbox: bool):
+    if use_sandbox:
+        MTURK_SANDBOX = "https://mturk-requester-sandbox.us-east-1.amazonaws.com"
```

Comment:
How do we determine `us-east-1` vs `us-west-1`? I think we should make this a param

Reply:
This has been implemented